### PR TITLE
feat: add collection banner

### LINKED ISSUE #5071: Hero Section layout
Tracked by
*  #5000 

## Hero section

* big change: hero section is becoming one big banner, with the name, social buttons and profile picture on top.
* there should be also option to upload this banner, issue already created, I will try to add designs as soon as possible.
* #4983 

image

*  banner on desktop has 1440*560, it would be also recommended size for the customization - its also used on collection cards. It has quite close aspect ration
* over banner is applied black gradient with lowered opacity, see figma
image

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5071

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

Screenshot 2023-02-22 at 19 31 31

### CODE REVIEW

File: components/collection/CollectionHeader/CollectionBanner.vue [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    if (metaName) {
+      collectionName.value = metaName
+    }
+
+    if (metaImage) {
+      collectionAvatar.value = metaImage
+    }
+  }
+})
+console.log(route.params.id, data.value)
```
Comment (Jarsen136) [inline components/collection/CollectionHeader/CollectionBanner.vue:62]:
```suggestion
```

File: components/collection/CollectionHeader/CollectionBanner.vue [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      width: 5.5rem;
+      height: 5.5rem;
+    }
+  }
+
+  &-name {
+    color: #fff;
+    text-shadow: 1px 1px 0 #000, 1px -1px 0 #000, -1px 1px 0 #000,
+      -1px -1px 0 #000, 1px 0 0 #000, 0 1px 0 #000, -1px 0 0 #000, 0 -1px 0 #000,
+      4px 4px #000;
```
Comment (Jarsen136) [inline components/collection/CollectionHeader/CollectionBanner.vue:103]: some color issue on dark/light mode 👀

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Review by roiLeo — APPROVED** [review]

**Comment by exezbcz** [comment]
Nice! 
image
Can you add please stroke to the heading? + drop shadow on collection pfp
thank you

**Comment by exezbcz** [comment]
yy, andd I see on the screenshot that the banner does not line up with navbar
image

**Comment by exezbcz** [comment]
@preschian 
hmmm, interesting solution, are there no other options left? :D
if no, it does not matter if its 84 or 85 :D

**Comment by exezbcz** [comment]
@preschian 
image
now the stroke is double stacked

**Comment by exezbcz** [comment]
yy, works good now, one thing is that there is still stroke missing on the heading
image

+ can you please remove left and right stroke as well? 
image


I think then it's going to be it,
thanks!

**Comment by vikiival** [comment]
@exezbcz is it ok?

**Review by vikiival — APPROVED** [review]
Very nice

**Comment by exezbcz** [comment]
@vikiival yes

**Comment by vikiival** [comment]
@preschian please resolve conflicts

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 37.29 USD/KSM ~ 0.805 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0x55de50e9107705a9051dcf1bcd0d556658cb22c54d9e923bfe3ed6109ed43045](https://kusama.subscan.io/extrinsic/0x55de50e9107705a9051dcf1bcd0d556658cb22c54d9e923bfe3ed6109ed43045)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**